app/presenters/work_presenter.rb

```diff
@@ -51,4 +51,8 @@ def publication_date
   private
 
   delegate :collection, :created_at, :user, to: :work
+
+  def license_presenter
+    @license_presenter ||= LicensePresenter.new(work_form: self, collection:)
```

Comment:
Is `self` here the `WorkPresenter` instance? It's being passed to the license presenter as the work form? Hmmm.

Reply:
The WorkPresenter is a SimpleDelegator that delegates to the work form, so this should be fine.

However, as you note, this is a bit off. I changed to get the work form from the work presenter.

Reply:
@justinlittman Excellent.